include/Options/Business/Force_Lang.php

```diff
@@ -28,6 +29,37 @@ public static function key(): string {
 		return 'force_lang';
 	}
 
+	/**
+	 * Adds information to the site health info array.
+	 *
+	 * @since 3.8
+	 *
+	 * @param array   $info    The current site health information.
+	 * @param Options $options An instance of the Options class providing additional configuration.
+	 *
+	 * @return array The updated site health information.
+	 */
+	public function add_to_site_health_info( array $info, Options $options ): array {
+		switch ( $options->get( self::key() ) ) {
```

Comment:
```suggestion
		switch ( $this->get() ) {
```
Let's use the current instance.